.fixtures.yaml

```diff
@@ -0,0 +1,10 @@
+fixtures:
+  forge_modules:
+    apt: 'puppetlabs/apt'
```

Comment:
usually we test against the master/main branches of dependencies in unit tests, but use their latest release in acceptance tests.

Reply:
easy enough, thanks